x-pack/plugins/siem/public/components/timeline/body/renderers/plain_column_renderer.tsx

```diff
@@ -22,16 +22,20 @@ import { parseQueryValue } from './parse_query_value';
 import { parseValue } from './parse_value';
 import { TruncatableText } from '../../../truncatable_text';
 
-import { IP_FIELD_TYPE } from './constants';
+import { IP_FIELD_TYPE, MESSAGE_FIELD_NAME } from './constants';
 
 export const dataExistsAtColumn = (columnName: string, data: TimelineNonEcsData[]): boolean =>
   data.findIndex(item => item.field === columnName) !== -1;
 
 const contextId = 'plain_column_renderer';
 
+// simple white-list to prevent dragging and dropping fields such as message name
```

Comment:
nit: this appears to be functioning as a blacklist, because items will be draggable by-default unless they are added to this (black)list

Reply:
Yea, it's a black list. I will change the wording.